Tests/Unit/EmogrifierTest.php

```diff
@@ -750,6 +756,43 @@ public function emogrifyFormatsCssDeclarations($cssDeclarationBlock, $expectedSt
     }
 
     /**
+     * Data provider for emogrifyInvalidDeclaration.
+     *
+     * @return array[]
```

Comment:
Oh, looks like I missed a pair of square brackets in my first comment. Sorry for the confusion I've caused. :confused: